util/ubuntu-dependencies

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This script installs streisand's dependencies on an Ubuntu 16.04
+# system; Debian jessie has been lightly tested. It installs system
+# packages for binary dependencies, and installs the rest in the
+# python directories in $HOME/.local.
+
+# We default to a regular, interactive upgrade:
+quiet=
+
+# It's safe to run this script multiple times.
+#
+# Please don't paste this directly into an interactive shell; if you
+# do, I will instantly close any bugs you file.
+#
+# If you're brave: this script can proceed mostly without interaction if:
+#
+# 1) "sudo echo" will succeed (i.e., you have root without a prompt now)
+#
+# 2) uncomment the following line:
```

Comment:
I think the grammar breaks slightly "without interaction if: <snip> 2) uncomment the following line:"